core/domain/platform_parameter_services.py

```diff
@@ -96,7 +96,7 @@ def get_server_mode() -> platform_parameter_domain.ServerMode:
     # TODO(release-scripts#137): Remove once project ID is verified on all
     # servers.
     logging.info('Logging project ID for debugging: %s' % (
-        os.environ['GOOGLE_CLOUD_PROJECT']))
+        os.environ.get('GOOGLE_CLOUD_PROJECT', 'dev-project-id')))
```

Comment:
dev-project-id is specifically for local dev, right ... should the default value be 'no-project-id-specified' or some other invalid value, to differentiate?

Reply:
Makes sense, updated.